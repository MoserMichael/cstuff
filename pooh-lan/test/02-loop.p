# loops: function computes the sum of squares over integer rane. 

sub squares( from, to)
  i = from
  res = 0
  while i < to
    res = res + i * i
    i = i + 1
  end
  return res
end

println( ~msg 'the sum of numbers is ' .. squares( ~from 1 ~to 7 ) )
