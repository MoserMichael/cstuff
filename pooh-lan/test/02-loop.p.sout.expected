the sum of numbers is 91
